tensorboard/plugins/profile/tf_profile_dashboard/tf-profile-dashboard.html

```diff
@@ -41,7 +41,25 @@
 <template is="dom-if" if="[[_dataNotFound]]">
   <div style="max-width: 540px; margin: 80px auto 0 auto;">
     <h3>No profile data was found.</h3>
-    <p>The dashboard is internal only; coming soon!</p>
+    <p>To collect a profile, you need to run your model on Google Cloud TPUs and
+    capture the trace information while your model is running. You may want to
+    check out the
+    <a href="https://github.com/tensorflow/tensorboard/blob/master/tensorboard/plugins/profile/README.md">README</a>
+    and perhaps the
+    <a
+     href="https://cloud.google.com/tpu/docs/cloud-tpu-tools"
```

Comment:
Just confirming that we're good to include this URLs in the repository now, even though they're not officially public yet?

Reply:
Confirmed.